### DISCUSSION TITLE
Issue with Creating New MonoRepo on Windows

### DISCUSSION THREAD

**Discussion by symaticvisuals** [shadcn-ui/ui#6198] [Q&A]
![image](https://github.com/user-attachments/assets/0b0f2da2-db56-4b4b-88b5-167ad77b869f)
![image](https://github.com/user-attachments/assets/b4daff14-7d33-4a7d-9bfc-262cf20457c2)

**Comment by akanlegumjohn** [comment]
I am also facing similar issue

**Comment by conradhunter** [comment]
works with pnpm & npm, seems to be bun issue

**Reply by akanlegumjohn** [reply]
I am using npm but I'm facing the same issue. These are the errors I see in my terminal, anything to help? 

Something went wrong. Please check the error below for more details.
If the problem persists, please open an issue on GitHub.
Command failed with exit code 1: npm install
npm ERR! code EUNSUPPORTEDPROTOCOL
npm ERR! Unsupported URL Type "workspace:": workspace:*

**Reply by morganbarber** [reply]
pnpm works.

**Comment by brasillero** [comment]
Exact same error using pnpm... I was wondering if it could be related to be under a proxy (in company), but it not seems to be the case.
